RushOrders/i18n/ko.json

```diff
@@ -0,0 +1,30 @@
+{
+    /*********
+    ** Shop UI
+    *********/
+    // Clint's shop
+    "clint.rush.name-suffix": "         *RUSH*",
```

Comment:
Can you translate `*RUSH*` and `=INSTANT=` too? Those are added to the item name in the shop menu for the option to finish the tool after one day or instantly. (There seems to be a bug where they're not shown anymore, but I'll look into that separately.)